src/test/resources/alma-fix/990185619180206441.json

```diff
@@ -12,6 +12,11 @@
     "location" : [ "s.l." ],
     "publishedBy" : [ "EMI Records" ]
   } ],
+  "manufacture" : [ {
+    "startDate" : "1994",
+    "type" : [ "Event" ],
+    "location" : [ "EMI Records Ltd, p 1969" ]
```

Comment:
This seems wrong. It should rather read "manufacturer" or similar and we should put the actual place of manufacture in this field.

Reply:
This is written in:
`$a - Place of production, publication, distribution, manufacture (R)`

this seems to be a cataloguing error.